docs/CHANGELOG.asciidoc

```diff
@@ -8,53 +8,641 @@
+Review important information about the {kib} 7.14.x releases.
+
+* <<release-notes-7.14.0>>
+//* <<release-notes-7.13.2>>
+//* <<release-notes-7.13.1>>
+//* <<release-notes-7.13.0>>
+//* <<release-notes-7.12.1>>
+//* <<release-notes-7.12.0>>
+//* <<release-notes-7.11.2>>
+//* <<release-notes-7.11.1>>
+//* <<release-notes-7.11.0>>
+//* <<release-notes-7.10.2>>
+//* <<release-notes-7.10.1>>
+//* <<release-notes-7.10.0>>
+//* <<release-notes-7.9.3>>
+//* <<release-notes-7.9.2>>
+//* <<release-notes-7.9.1>>
+//* <<release-notes-7.9.0>>
+//* <<release-notes-7.8.1>>
+//* <<release-notes-7.8.0>>
+//* <<release-notes-7.7.1>>
+//* <<release-notes-7.7.0>>
+//* <<release-notes-7.6.2>>
+//* <<release-notes-7.6.1>>
+//* <<release-notes-7.6.0>>
+//* <<release-notes-7.5.2>>
+//* <<release-notes-7.5.1>>
+//* <<release-notes-7.5.0>>
+//* <<release-notes-7.4.2>>
+//* <<release-notes-7.4.1>>
+//* <<release-notes-7.4.0>>
+//* <<release-notes-7.3.2>>
+//* <<release-notes-7.3.1>>
+//* <<release-notes-7.3.0>>
+//* <<release-notes-7.2.1>>
+//* <<release-notes-7.2.0>>
+//* <<release-notes-7.1.1>>
+//* <<release-notes-7.1.0>>
+//* <<release-notes-7.0.1>>
+//* <<release-notes-7.0.0>>
+//* <<release-notes-7.0.0-rc2>>
+//* <<release-notes-7.0.0-rc1>>
+//* <<release-notes-7.0.0-beta1>>
+//* <<release-notes-7.0.0-alpha2>>
+//* <<release-notes-7.0.0-alpha1>>
 
 --
 
+[[release-notes-7.14.0]]
+== {kib} 7.14.0
+
+coming::[7.14.0]
+
+For information about the {kib} 7.14.0 release, review the following information.
+
+[float]
+[[breaking-changes-7.14.0]]
+=== Breaking changes
+
+Breaking changes can prevent your application from optimal operation and performance.
+Before you upgrade to 7.14.0, review the breaking changes, then mitigate the impact to your application.
+
+// tag::notable-breaking-changes[]
+
+[discrete]
+[[breaking-102263]]
+.Changes audit event terminology
+[%collapsible]
+====
+*Details* +
+The terminology has changed for audit events logged by the alerting framework alerts client when CRUD operations are performed on rules. 
+
+The following audit actions are renamed:
+
+* `alert_create` is now `rule_create`
+
+* `alert_get` is now `rule_get`
+
+* `alert_update` is now `rule_update`
+
+* `alert_update_api_key` is now `rule_update_api_key`
+
+* `alert_enable` is now `rule_enable`
+
+* `alert_disable` is now `rule_disable`
+
+* `alert_delete` is now `rule_delete`
+
+* `alert_find` is now `rule_find`
+
+* `alert_mute` is now `rule_mute`
+
+* `alert_unmute` is now `rule_unmute`
+
+* `alert_instance_mute` is now `rule_alert_mute`
+
+* `alert_instance_unmute` is now `rule_alert_unmute`
+
+The terminology in the audit messages now references `rules` instead of `alerts`. For example, `User is creating alert [id=${ruleId}]` is now `User is creating rule [id=${ruleId}]`.
+
+For more information, refer to {kibana-pull}102263[#102263].
+
+*Impact* +
+When you upgrade to 7.14.0, {kib} automatically uses the new terminology. No action is needed.
+====
```

Comment:
@ymao1 does this "changes audit event terminology" section look good to you for the 7.14 release notes? This is copied from your PR: https://github.com/elastic/kibana/pull/102263.

Reply:
Yep, sounds accurate to me!